compiler/noirc_errors/src/reporter.rs

```diff
@@ -16,6 +16,7 @@ pub struct CustomDiagnostic {
 pub enum DiagnosticKind {
     Error,
     Warning,
+    Debug,
```

Comment:
Can we rename this to `Note` since they are rendered as notes eventually?

Reply:
`Info` would make most sense as it would line up with the LSP types.

Reply:
Ah, I was thinking `Note` because they're rendered as notes to our error library on line 201, but either would be an improvement over Debug I think.

Reply:
Renamed to `Info`